workflow/rocoto/gfs_tasks.py

```diff
@@ -86,10 +86,10 @@ def prep(self):
         gfs_enkf = True if self.app_config.do_hybvar and 'gfs' in self.app_config.eupd_cdumps else False
 
         deps = []
-        dep_dict = {'type': 'metatask', 'name': 'gdaspost', 'offset': '-06:00:00'}
+        dep_dict = {'type': 'metatask', 'name': 'gdaspost', 'offset': self._base['minus_cycle_interval']}
```

Comment:
create the `minus` here with appropriate methods.
```suggestion
        dep_dict = {'type': 'metatask', 'name': 'gdaspost', 'offset': f"-{timedelta_to_HMS(self._base['cycle_interval'])}"}
```